builtInFunctions/creator.go

```diff
@@ -138,6 +138,15 @@ func (b *builtInFuncCreator) CreateBuiltInFunctionContainer() error {
 		return err
 	}
 
+	newFunc, err = NewMigrateUserNameFunc(b.gasConfig.BuiltInCost.SaveUserName, b.mapDNSAddresses, b.enableEpochsHandler)
+	if err != nil {
+		return err
+	}
+	err = b.builtInFunctions.Add("migrateUserName", newFunc)
```

Comment:
Can we add a const here to keep consistency with other builtInfFuncs ?

Reply:
const added.